secure-import/README.md

```diff
@@ -27,4 +27,7 @@ A lightweight tool to encrypt nonce with AES-GCM or AES-CBC encryption on a nonc
     {
         "encryptedNonce": "DVy/Dbk37X8gSVwRA5U6vrHdWQy8T2ej+riIVw==",
         "iv": "puQrzDX7gU1TcTTx"
-    }
\ No newline at end of file
+    }
+    
```

Comment:
On line 31, does there need to be a closing "fence" (\`\`\`) for the code block after the example?